classes/blocks/class-covers.php

```diff
@@ -217,6 +220,39 @@ private static function filter_posts_for_act_pages( $fields ) {
 		return [];
 	}
 
+	/**
+	 * Get posts that are Action pages (p4_action).
+	 *
+	 * @param array $fields This is the array of fields of this block.
+	 *
+	 * @return \WP_Post[]
+	 */
+	private static function filter_posts_for_action_pages( $fields ): array {
+		$tag_ids = $fields['tags'] ?? [];
+		$layout  = $fields['layout'] ?? self::GRID_LAYOUT;
+
+		$args = [
+			'post_type'        => ActionPage::POST_TYPE,
+			'post_status'      => 'publish',
+			'orderby'          => [
+				'date'  => 'DESC',
+				'title' => 'ASC',
+			],
+			'suppress_filters' => false,
+			'numberposts'      => self::CAROUSEL_LAYOUT === $layout
```

Comment:
Not really related to this ticket, but we define this `numberposts` several times using the exact same line of code, maybe worth creating a small function for it?